### PR TITLE
add search fix to user facing changes

### PR DESCRIPTION
**Pull request by lokijuhy**
* note the search fix in the user facing changes section
* some minor wordsmithing

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -3,6 +3,10 @@
 0.58.0
 ------
 
+Renku ``0.57.2`` fixes several issues related to Renku 2.0 search, and also squashes a bug where the
```
Comment (Panaetius) [inline CHANGELOG.rst:6]:
```suggestion
Renku ``0.58.0`` fixes several issues related to Renku 2.0 search, and also squashes a bug where the
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -12,9 +16,10 @@ User-Facing Changes
 
 **🐞 Bug Fixes**
 
+- **Search Services**: Resolve issues that caused items to be missing from Renku 2.0 search, including the in the search for adding members to projects and groups.
```
Comment (Panaetius) [inline CHANGELOG.rst:19]:
```suggestion
- **Search Services**: Resolve issues that caused items to be missing from Renku 2.0 search, including in the search for adding members to projects and groups.
```

**Review by Panaetius — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (lokijuhy) [reply CHANGELOG.rst:6]: 🙈 oh my gosh thank you!

**Review by Panaetius — APPROVED** [review]

File: CHANGELOG.rst (hunk 2 above)
Comment (m-alisafaee) [reply CHANGELOG.rst]: I cannot process the second part of the sentence; do we want to give examples of items that were missing or we want to say that search now includes projects and groups' members. If the first case, I'd write it as `..., including  projects and groups' members` and if the second case: `... including adding members to projects and groups in the search`.

**Review by Panaetius — APPROVED** [review]

**Pull request merged**